src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -59,26 +61,45 @@ export default class Browser extends DashboardView {
 
       lastError: null,
       relationCount: 0,
+
+      attachRowsErrors: null,
     };
+
+    this.showAttachRowsDialog = this.showAttachRowsDialog.bind(this);
+    this.cancelAttachRows = this.cancelAttachRows.bind(this);
+    this.confirmAttachRows = this.confirmAttachRows.bind(this);
   }
 
   componentWillMount() {
     this.props.schema.dispatch(ActionTypes.FETCH)
     .then(() => this.handleFetchedSchema());
+
+    const { className, entityId, relationName } = this.props.params;
+
     if (!this.props.params.className && this.props.schema.data.get('classes')) {
       this.redirectToFirstClass(this.props.schema.data.get('classes'));
     } else if (this.props.params.className) {
+      let filters = new List();
       if (this.props.location.query && this.props.location.query.filters) {
-        let filters = new List();
```

Comment:
if so it's declared in the wrong place as you exit the `if` block and expect the filters still to be set